js/render.js

```diff
@@ -418,36 +426,23 @@ function render(hash, options) {
         ]);
     }
 
-    function updateHash(osmType, featureId) {
-      clearHash();
-
-      location.hash += '/' + osmType;
-      location.hash += '/' + featureId;
-    }
-
-    function clearHash() {
-      var changesetId = location.hash
-        .split('/')[0]
-        .replace('#', '');
-
-      location.hash = changesetId;
-    }
-
     function selectFeature(feature, featureMap) {
       var featureId = feature.properties.id;
       var osmType = feature.properties.type;
 
       highlightFeature(featureId);
       displayDiff(featureId, featureMap);
-      updateHash(osmType, featureId);
+      cmap.emit('hashchange', osmType, featureId);
```

Comment:
@geohacker can we call this event just `change`? Since the module is not directly speaking to the location hash, `hashchange` doesn't seem to be quite accurate here.